packages/server/controllers/events.js

```diff
@@ -121,6 +121,29 @@ async function getEventAttendees(eventId) {
   return query;
 }
 
+async function updateEventAttendeeStatus(userId, eventId, status) {
+  // Validate status
+  const allowedStatusValues = ["NotInterested", "Interested", "Attending"];
+  if (!allowedStatusValues.includes(status)) {
+    throw new Error(
+      "Invalid status value. Allowed values are: NotInterested, Interested, Attending"
+    );
+  }
+
+  const updatedEventAttendee = await prisma.eventAttendees.update({
+    where: {
+      userId_eventId: {
+        userId: userId,
+        eventId: eventId,
+      },
+    },
+    data: {
+      status: status,
+    },
+  });
```

Comment:
You should use [upsert](https://www.prisma.io/docs/orm/prisma-client/queries/crud#update-or-create-records) instead of update because this will fail if there is no existing record yet when trying to update.